client/Components/Decks/DeckSummary.jsx

```diff
@@ -157,6 +163,7 @@ class DeckSummary extends React.Component {
                         { cardsToRender }
                     </div>
                 </div>
+                <div style={ divStyle } />
```

Comment:
This should be in the .css/.less if it's not dynamic

Reply:
Ok, I will fix it.